samples/v1.5/Elements/SecondaryActions.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.2",
+  "body": [
+	{
+	  "type": "Image",
+	  "url": "https://adaptivecards.io/content/adaptive-card-50.png"
+	},
+	{
+	  "type": "TextBlock",
+	  "text": "Hello **Adaptive Cards!**"
+	},
+	{
+	  "type": "ActionSet",
+	  "actions": [
+		{
+		  "type": "Action.Submit",
+		  "title": "Action Submit",
+		  "data": {
+			"key": "Submit from overflow menu"
+		  }
+		},
```

Comment:
did you mean to mark this as secondary, or should the key read `Submit from action button`?

Reply:
This is an example to show primary & secondary actions together. If mode is not specific, it will be considered as "primary" button and "data" will be passed on tapping the buttons.